tests/test_pyfar_base.py

```diff
@@ -0,0 +1,45 @@
+from pyfar.classes._pyfar_base import _PyfarBase
+
+class Dummy(_PyfarBase):
+    """
+    Minimal dummy class for testing _PyfarBase.
+    """
+
+    def _decode(self):
+        pass
+
+def test_comment():
+    """Test the comment property."""
+
+    instance = Dummy("Dummy #1")
+    assert instance.comment == "Dummy #1"
+
+def test_comment_setter():
+    """Test the comment.setter."""
+
+    instance = Dummy()
+    instance.comment = "Dummy #2"
```

Comment:
```suggestion
    assert instance.comment == ''
    instance.comment = "Dummy #2"
```
in this way we could check the default